src/SFA.DAS.ApprenticeCommitments.Web.UnitTests/Features/ConfirmIdentityEnforcedSteps.cs

```diff
@@ -79,7 +79,7 @@ public void ThenRedirectTheUserToTheHomePage()
         [Then("redirect the user to the TermsOfUse page")]
         public void ThenRedirectTheUserToTermsOfUse()
         {
-            _context.Web.Response.Should().Be302Found();
+            _context.Web.Response.StatusCode.Should().Be(HttpStatusCode.Redirect);
```

Comment:
Using [FluentAssertions.Web](https://github.com/adrianiftode/FluentAssertions.Web/#examples) provides much more information if the tests fail, and should be preferred over directly testing the `HttpStatus` value
```suggestion
            _context.Web.Response.Should().Be302Redirect();
```